jquery.marquee.js

```diff
@@ -121,8 +121,10 @@
                 }
             });
 
-            // since speed option is changed to duration, to support speed for those who are already using it
-            o.duration = o.speed || o.duration;
+            // Reintroduce speed as an option. It calculates duration as a factor of the container width
+            if (o.speed) {
```

Comment:
It'd certainly be a different value to what they set if they had defined a speed also, otherwise, I'd assume if they were just upgrading, they wouldn't define a speed, and duration would continue to work correctly.